server/config.ts

```diff
@@ -140,9 +140,12 @@ export const auth = {
   },
 };
 
-export const redis = {
-  uri: envVar({ name: "REDIS_URI_CACHE", defaultValue: "" }),
-  username: envVar({ name: "REDIS_USERNAME_CACHE", defaultValue: "" }),
-  password: envVar({ name: "REDIS_PASSWORD_CACHE", defaultValue: "" }),
+export const valkey = {
+  uri: envVar({ name: "VALKEY_URI_CACHE", defaultValue: "" }).replace(
+    "valkeys",
+    "rediss"
```

Comment:
klient-rammeverket vi bruker her kjenner ikke til protokollen valkeys (som nais legger inn uri'en), så bruker rediss i stedet.

Reply:
Vet ikke om jeg skjønte helt hva dette betyr?

Reply:
URI'en for å lage en connection mot Aiven Redis/Valkey ser slik ut: 
rediss://<user>:<password>@valkey-teamsykefravr-cache-nav-prod.l.aivencloud.com:26483

Protokollen "rediss" er noe som Jedis-rammeverket (som er klient-rammeverket vi bruker) skjønner. Det er tydeligvis også laget en variant som kalles "valkeys" som Nais dytter inn i env-variabelen VALKEY_URI_CACHE, men dette er ikke noe som Jedis skjønner. Så bytter dette tilbake til "rediss" som også fungerer bra mot Valkey-cachen.

Reply:
I Ktor/Kotlin-app'ene oppretter vi denne forbindelsen på en annen måte og bruker env-variablene annerledes, så der støter vi ikke på det samme problemet.